### DISCUSSION TITLE
Velocyto in sc-RNAseq coupled TCRseq on 5'

### DISCUSSION THREAD

**Discussion by Baldosisley** [theislab/scvelo#1250] [General]
Greetings everyone,

I recently conducted an experiment combining sc-RNAseq with TCRseq using a 5' approach. After running velocyto and loading the .loom files, everything seemed to proceed smoothly. However, upon plotting the pie chart of spliced and unspliced reads, I noticed an unusual distribution: 64% unspliced, 32% spliced, and 4% ambiguous.

This finding is unexpected, as the scVelo vignette and the literature I've consulted typically report inverted percentages of spliced versus unspliced reads. Therefore, I'm curious about the feasibility of conducting this analysis with a 5' GEX and TCR approach. Is it possible, or does it require different preprocessing methods?

I would greatly appreciate any recommendations or insights you might have regarding this type of analysis.

Thank you sincerely.

**Comment by DoaneAS** [comment]
Hi,

I've also seen this in 5' GEX data, using velocyto to generate loom files from 10x output.  I'm wondering if velocyto is handling the unspliced read calling in a way that makes sense for 5' GEX.

**Comment by OlgaLanzetta** [comment]
Hi @DoaneAS and @Baldosisley 

I am facing the exact same issue with my 5' GEX (R2-only) dataset, where velocyto is outputting an incredibly high fraction of unspliced reads (around 58%).

As you suspected, velocyto doesn't seem to handle the strandedness/direction of 5' data correctly out of the box, as it was originally optimized for 3' chemistries. A recent 2025 preprint actually confirmed that velocyto systematically misassigns spliced reads to the unspliced/reverse strand category in 5' data.

Have you or anyone else found a reliable solution or workaround for this? Did you manage to fix the counts by changing the velocyto parameters (like the strandedness flags), or did you switch to alternative quantification tools like alevin-fry or Tidesurf?

Thanks!
